src/main/webapp/app/localci/build-queue/build-queue.component.html

```diff
@@ -660,3 +682,131 @@ <h3 id="build-queue-finished-heading" jhiTranslate="artemisApp.buildQueue.finish
         </div>
     }
 </div>
+<!-- Modal -->
+<ng-template #content let-modal>
+    <div class="modal-header">
+        <h5 class="modal-title">{{ 'artemisApp.buildQueue.filter.title' | artemisTranslate }}</h5>
+        <button type="button" class="btn-close" aria-label="Close" (click)="modal.dismiss()"></button>
+    </div>
+    <div class="modal-body">
+        <div class="container-fluid">
+            <div class="row finished-build-jobs-filter-border-bottom">
+                <div class="col d-flex flex-row pb-1">
+                    <h5 class="my-0">{{ 'artemisApp.buildQueue.filter.buildStatus.title' | artemisTranslate }}</h5>
+                </div>
+            </div>
+            <div class="row finished-build-jobs-filter-background-accent">
+                <div class="col container px-3">
+                    <ul class="text-nowrap list-unstyled row my-0 py-2">
+                        @for (status of this.buildJobStatusFilter; track status) {
+                            <li class="col">
+                                <label>
+                                    <input
+                                        class="form-check-input"
+                                        (change)="this.toggleBuildStatusFilter(status)"
+                                        [checked]="this.finishedBuildJobFilter.status === status"
+                                        type="radio"
+                                    />
+                                    <span class="ps-1">{{ 'artemisApp.buildQueue.filter.buildStatus.' + status.toString().toLowerCase() | artemisTranslate }}</span>
```

Comment:
1. Could we use `jhiTranslate` here?
2. We should avoid function calls in the template, as they are called over and over again by the change detection. Can you pre-compute this information somehow?